src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -559,7 +559,8 @@ CHIP_ERROR P256PublicKey::ECDSA_validate_hash_signature(const uint8_t * hash, co
 
 CHIP_ERROR P256Keypair::ECDH_derive_secret(const P256PublicKey & remote_public_key, P256ECDHDerivedSecret & out_secret) const
 {
-#if defined(MBEDTLS_ECDH_C) && !defined(MBEDTLS_ECP_ALT)
+// TODO: Enable ECDH_derive_secret for Qorvo when their mbedTLS static libraries are updated
```

Comment:
@tima-q FYI. I must have kept it disabled for Qorvo for a very similar reason as mentioned in https://github.com/project-chip/connectedhomeip/pull/7311. How is the status of updating Qorvo's mbedTLS libraries?

Reply:
there is a PR available on Lukas' PR branch - updating our lib - #7311
 https://github.com/LuDuda/connectedhomeip/pull/17

Reply:
Great, thanks. Lukas is not available this week, but I've merged your changed.